.gitignore

```diff
@@ -1 +1,2 @@
-script/node_modules/
+server/sendgrid/
+server/sengrid.mail/
```

Comment:
I believe we should keep _script/node_modules/_ and also looks like those added lines won't work. Please read about _.gitignore_ patterns and examples: https://git-scm.com/docs/gitignore#_pattern_format

Reply:
in folder _script_ I do not have a _node_modules_. 
In the _script_, I did not use _sendgrid_. I delete this file(_node/modules_), and downloaded the _sendgrid_ and keep it in the folder in the server folder.